qcommon/src/org/qcmg/common/model/SmithWatermanGotoh.java

```diff
@@ -4,7 +4,7 @@
  *
  * This code is released under the terms outlined in the included LICENSE file.
  */
-package org.qcmg.qmule;
+package org.qcmg.common.model;
```

Comment:
consider a different location within qcommon, model is meant for POJOs

Reply:
really not sure where to fit it in. How about org.qcmg.common.utils? alternatively, we can modify SmithWatermanGotoh.java to make it to be POJO class.  I also guess this class is still in development, it may move again. 
anyway, it is up to your suggestion since you are the owner of it.

Reply:
how about putting it into its own package? `sw` sounds good

Reply:
yes, all good now!